packages/autocomplete-plugin-recent-searches/src/usecases/localStorage/search.ts

```diff
@@ -10,12 +34,13 @@ export function search<TItem extends RecentSearchesItem>({
   query,
   items,
   limit,
-}: SearchParams<TItem>) {
+}: SearchParams<TItem>): Array<Highlighted<TItem>> {
   if (!query) {
-    return items.slice(0, limit);
+    return items.map((item) => highlight({ item, query })).slice(0, limit);
```

Comment:
no so much of difference, but we can highlight the array AFTER slicing it to the smaller size. (the same below)